server.js

```diff
@@ -51,15 +57,16 @@ app.locals.asset_path = '/public/'
 app.locals.useAutoStoreData = (useAutoStoreData === 'true')
 app.locals.useCookieSessionStore = (useCookieSessionStore === 'true')
 app.locals.releaseVersion = 'v' + releaseVersion
-app.locals.serviceName = config.serviceName
+app.locals.serviceName = config.serviceName || 'GOV.UK Prototype Kit'
 // extensionConfig sets up variables used to add the scripts and stylesheets to each page.
 app.locals.extensionConfig = extensions.getAppConfig()
+app.locals.extensionConfig.scripts.push('/assets/javascripts/application.js')
```

Comment:
For consistencies sake, is it worth making a similar change to how we load `application.scss`?

Reply:
Yes, I've made the "proper" change I wanted to make at the time but didn't want to get distracted by.  We can now feed user scripts/stylesheets in as a parameter to `getAppConfig()`.  This is now used to include both scripts and stylesheets that are in the user's kit.